packages/dds/tree/src/tableSchema.ts

```diff
@@ -911,6 +931,10 @@ export namespace System_TableSchema {
  * });
  * ```
  *
+ * @privateRemarks
+ * The above examples are backed by tests in `tableSchema.spec.ts`.
+ * Those tests and these examples should be kept in-sync to ensure that the examples are correct.
+ *
```

Comment:
```suggestion
 *
 * TODO: Transaction constraint APIs should be added which make it possible for a user
 * to be able to opt into preventing the cell leaking mentioned above, at which point those docs should be updated.
```

Reply:
I'll add a comparable note elsewhere in the file. Since the implementation is split across namespaces, and because `@privateRemarks` blocks are visible in intellisense, I am tracking wider-scoped TODOs at the top of the module.